python/py-libtmux/Portfile

```diff
@@ -20,9 +20,9 @@ homepage            https://github.com/tmux-python/libtmux
 master_sites        pypi:l/${python.rootname}/
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  d0434b68e84b6572fd149eb4be24ad9118d423b0 \
-                    sha256  2acd86617203bc966b125d016526fb40503411a2281b55d9d970a487dc6e065a \
-                    size    37725
+checksums           rmd160  ace9b7341b48f683911bece86bf6040270d498e9 \
+                    sha256  1d35b9f8451944d31c5ed22ed9e6c8e18034adcc75718fcc5b27fbd9621543e1 \
+                    size    36961
 
 python.versions     27 35 36 37 38
```

Comment:
We can also take this opportunity to add Python 3.9 support here, by appending `39` to `python.versions`.

Reply:
Tested and done.